src/utils/command.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Helper function to get the command arguments
+ *
+ * @param {string} cmd - Command name
+ * @param {string} message - The original message
+ * @return {string} command argument
+ */
+export const getCommandArgs = (cmd, context) => {
+  const { message, me } = context;
```

Comment:
`message` here is an object. the text resides in `context.message.text`. unless you've check it somehow?

Reply:
oh iya